RealmBrowser/RealmBrowser-Info.plist

```diff
@@ -42,7 +42,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>8</string>
```

Comment:
Because the build number of the current version of the Browser on the App Store is 7.